lib/class-wp-rest-plugins-controller.php

```diff
@@ -194,7 +249,16 @@ public function get_item_schema() {
 	}
 
 	public function get_collection_params() {
-		return array();
+		$params = parent::get_collection_params();
+
+		$params['offset'] = array(
```

Comment:
Oh, as `page` and `per_page` are already brought in from `parent::get_collection_params()` then that's fine. As for `offset`, I hadn't realized we also support that in the other controllers too. Who knows what happens if you specify `offset` with `page`!